Parameters

SchulzJonathan opened the pull request:
Port Settings can now be changed using Parameters, e.g.by typing
```
ros2 param set /DSRC_Handler port <new_port>
```
in the Terminal.

The Ports can now be changed for the DSRC as well as the GPS Publisher. The initial Settings of the Parameters are the same as hardcoded before, therefore this is not a breaking change. The implemented Parameters as well as their initial Values are listed in the Table below.

Parameter | Initial Value
---|---
/DSRC_Handler port | 37008
/GPS_Publisher port | 37010

Commit:
feat: parameters for the DSRC Publisher.

The Port can now be modified using Parameters.

Commit:
feat: Added Port setting for DSRC Publisher.

- Port can now be set using ros2 param set /DSRC_Handler port <port>

EliasRothfuss commented on v2x_cohdatoros/UDPService.py:
```diff
@@ -1,13 +1,10 @@
+import rclpy
```
Warum?

EliasRothfuss commented on v2x_cohdatoros/GPS.py:
Ich arbeite inzwischen mit der NavSatFix Message für GPS, evtl. direkt anpassen.
https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/NavSatFix.html

EliasRothfuss reviewed (commented):
Geh mal davon aus, dass der Rest funktioniert

SchulzJonathan committed:
feat! Switch to NavSatFix Message and Bugfixes

- Switched from the custom GPS Message to NavSatFix Message
- Removed import rclpy from UDP Service
- Fixed a bug where error Messages weren't converted to string and
  therefore lead to a new error
